packages/cli/src/evaluation.ee/test-runs.controller.ee.ts

```diff
@@ -65,7 +65,7 @@ export class TestRunsController {
 		}
 	}
 
-	@Get('/:workflowId/test-runs/:id/cases')
+	@Get('/:workflowId/test-runs/:id/test-cases')
```

Comment:
I would expect integration test related changes here. Would be nice to add integration tests here.

Also I expect some FE changes, but cannot find relevant changes here. Is that already merged in with a different PR?

Reply:
There are in `packages/cli/test/integration/evaluation/test-runs.api.test.ts`, I added tests for this endpoint and some others.

Frontend changes will follow right after this PR is merged into master: https://github.com/n8n-io/n8n/pull/15550

Reply:
thanks for adding those